File: BackEnd/IBMYoung/Model/Resposta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IBMYoung.Model {
    public class Resposta {
        public Aprendiz Aprendiz { get; set; }
        public int AprendizId { get; set; }
        public Questao Questao { get; set; }
        public int TarefaId { get; set; }
        public int Ordem { get; set; }
        public Alternativa Alternativa { get; set; }
        public int AlternativaId { get; set; }
    }
}

File: Mobile/IBM_Young/app/src/main/java/br/senai/sp/informatica/ibmyoung/model/Resposta.java
package br.senai.sp.informatica.ibmyoung.model;

/**
 * Created by pena on 02/04/2018.
 */

public class Resposta {
    private int alternativaId;

    public Resposta() {
    }

    public Resposta(int alternativaId) {
        this.alternativaId = alternativaId;
    }

    public int getAlternativaId() {
        return alternativaId;
    }

    public void setAlternativaId(int alternativaId) {
        this.alternativaId = alternativaId;
    }
}
